<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChatList</class>
 <widget class="QWidget" name="ChatList">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>606</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>930</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>群聊列表</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#frame{
border: 1px solid rgb(41, 57, 85);
border-radius: 10px;
border-color:#dcdee0;
background-color: rgba(255, 255, 255,0.9);
}
QLabel{
color:rgb(0, 0, 0)
}
QPushButton {
background-color:#030408;
color:#db9a45;
font:9pt &quot;Microsoft YaHei UI&quot;;
border-radius:5px;
font-weight:650;
height:25px;
width: 80%;
}
QPushButton:disabled {
background-color: #787a7e;
}
QPushButton:hover {
background-color: #383a3e;
}
QListWidget{
background:rgba(255,255,255,0.7);
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QFrame" name="frame">
     <property name="maximumSize">
      <size>
       <width>500</width>
       <height>930</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">#frame{border-image:url(pic/listback3.jpg)}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="UserNameLabel">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="closeButton">
          <property name="maximumSize">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">#closeButton{
border:none;
background:rgba(255,255,255,0.4);
}
#closeButton:hover {
background: rgba(255,255,255,0.7);
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="HJoinChat">
            <property name="text">
             <string>已加入的群聊</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="OpenChatButton">
            <property name="text">
             <string>打开群聊</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QListWidget" name="HJoinChatListWidget"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLineEdit" name="ChatNameInput">
            <property name="styleSheet">
             <string notr="true">QLineEdit{
border: 1px solid rgb(41, 57, 85);
border-radius: 3px;
border-color:#dcdee0;
background: rgba(255,255,255,0.6);
padding-left: 12px;
padding-right: 12px;
border-color:#e4e5eb;
}
QLineEdit:hover {
border-color: #db9a45;
}</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="FindChatButton">
            <property name="text">
             <string>搜索</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="NewChatButton">
            <property name="text">
             <string>创建新群聊</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="UJoinChat">
            <property name="text">
             <string>未加入的群聊</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="JoinButton">
            <property name="text">
             <string>加入</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QListWidget" name="UJoinChatListWidget"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
